rln/rln.go

```diff
@@ -113,23 +117,45 @@ func toIdentityCredential(generatedKeys []byte) (*IdentityCredential, error) {
 
 // MembershipKeyGen generates a IdentityCredential that can be used for the
 // registration into the rln membership contract. Returns an error if the key generation fails
-func (r *RLN) MembershipKeyGen() (*IdentityCredential, error) {
+// Accepts an optional parameter that sets the user message limit which defaults
+// to DEFAULT_USER_MESSAGE_LIMIT
+func (r *RLN) MembershipKeyGen(params ...uint32) (*IdentityCredential, error) {
```

Comment:
I wonder if it wouldnt be better to rename `params` to userMessageLimit`?

Reply:
indeed 5bd2a083d16a5613b5e273776ef69927f2c1f1ff